### ISSUE TITLE
`title` attribute is missing on <iframe> tag suggested for embedding

### ISSUE THREAD

**Issue by lunar-debian** [Chocobozzz/PeerTube#3861]
**Describe the current behavior**
When I want to embed a video in another webpage, the generated code contains an `<iframe>` tag without a `title` attribute. The latter being recommended for accessibility purposes.

**Steps to reproduce:**

1. Open the page of a video.
2. Click “Share”
3. Click “Embed”

The generated HTML code is missing the `title` attribute in the `<iframe>` tag.

**Describe the expected behavior**
The `title` attribute should be set with the title of the video.

**Additional information**

* PeerTube instance:
  * URL: https://aperi.tube/
  * version: 3.0.1

This is probably a nice issue to be solved by newcomers. :)

**Comment by rigelk** [comment]
I guess https://www.w3.org/TR/WCAG20-TECHS/H64.html is the recommendation you refer to?

**Comment by pthavarasa** [comment]
Hi @rigelk,

I fixed this issue. Now there is a title in the iframe tag.
share video -> video title
share playlist -> playlist title
share playlist video(right click in the video to copy embed) -> video title

**Comment by rigelk** [comment]
@pthavarasa please submit a PR then

**Comment by pthavarasa** [comment]
@rigelk i have no idea if i made a proper PR, can you help me please?
